website/content/docs/agent/index.mdx

```diff
@@ -366,6 +366,63 @@ service {
 </Tab>
 </Tabs>
 
+## Client Node with Multiple Interfaces or IP addresses
+
+The following example shows how to configure Consul to listen on multiple interfaces or IP addresses using a [go-sockaddr template].
+
+The `bind_addr` is used for internal RPC and Serf communication (see the Agent Configuration for [bind_addr](/docs/agent/options#bind_addr) for more information).
+
+The `client_addr` configuration specifies IP addresses used for HTTP, HTTPS, DNS and GRPC servers. (see the Agent Configuration for [client_addr](/docs/agent/options#client_addr) for more information).
```

Comment:
Could you make the link name more accessible? A screen-reader won't be able to clarify the context of where the link is sending the reader, if they use a screen reader.

Ref: https://www.washington.edu/accessibility/checklist/links/

Reply:
Easiest way would be to expand the link text to cover more of the sentence here?

Reply:
Yeah. Something like "Read [the Agent Configuration options page](link) for more information." This gets read by a screen reader as: "LINK: the Agent Configuration options page". Replacing "Read" with "see", removes the context of how to consume the content, as in this case the the accessible link description could be for folks with different visual acuity.

Reply:
I gave it a shot.